src/worker.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+var queue = require('queue-async');
+var map = require(process.argv[2]);
+
+var q = queue();
+var sources = [];
+
+JSON.parse(process.argv[3]).forEach(function(source) {
+  q.defer(loadSource, source);
+});
+
+function loadSource(source, done) {
+  if (source.mbtiles) sources.push({name: source.name, getTile: require('./mbtiles')(source, done)});
```

Comment:
Ok, I am going to proceed with writing tests for this in its current form and may refactor in a separate PR (requires some refactoring of the worker implementation). The remote interface  diverts the execution flow in a confusing way IMO, for example:

```js
var getTile = remote(source, function(err) {
  // i need to wait until i get here to see if there is an error on initialization
  if(err) throw err;
  // next line will fail because getTile has not actually returned yet, despite the parser being "ready"
  getTile([5276, 12757, 15], function(err, layers) {
    // yay, results!
  });
});
```